aspnetcore/fundamentals/minimal-apis/includes/results-helpers.md

```diff
@@ -0,0 +1,22 @@
+Common result helpers exist in the `Microsoft.AspNetCore.Http.Results` static class.
+
+| Description                                                                     | Response type                                    | Status Code                 | API                                                                                          |
+| ------------------------------------------------------------------------------- | ------------------------------------------------ | --------------------------- | -------------------------------------------------------------------------------------------- |
+| Write a JSON response with advanced options.                                    | application/json                                 | 200                         | [Results.Json](xref:Microsoft.AspNetCore.Http.Results.Json%2A)                               |
```

Comment:
What does "advanced options" mean here? What would be examples of non-advanced options? If "advanced" refers to customizing serialization, maybe it would be clearer to say "Write a JSON response with customized serialization options." And then Results.Ok would be "Write a JSON response with default serialization options."

Reply:
@tdykstra I changed to:

**OK** -> Set the status code to 200, with an optional JSON response.
**JSON** -> Write a JSON response with optional serialization options.